lib/generators/onesie/template_generator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails/generators/named_base'
+
+module Onesie
+  module Generators
+    # Generates a new Onesie template
+    class TemplateGenerator < Rails::Generators::NamedBase
```

Comment:
Can you add documentation in the `how_to_guides/usage.md` about this generator please

Reply:
Can do!

Reply:
@timlkelly Done!